onnxruntime/core/optimizer/insert_cast_transformer.cc

```diff
@@ -257,7 +259,7 @@ class RemoveDuplicateCastTransformer : public GraphTransformer {
             }
 
             // Cannot remove node if it's output is also an output of the graph
-            if (graph_outputs.find(output_node.OutputDefs()[0]) == graph_outputs.end() &&
+            if (graph_outputs.find(output_node.OutputDefs()[0]) == graph_outputs_end &&
```

Comment:
>find [](http://example.com/codeflow?start=30&length=4)

!contains()

Reply:
I will change these places, but it is not a part of C++ until C++20 and we may decided to change the library.